### DISCUSSION TITLE
Additional number formats

### DISCUSSION THREAD

**Discussion by tht** [home-assistant/frontend#9432] [Localization]
I just realized that it is possible to change the number format in my user profile in Home-Assistant (core-2021.6.5) :).

IMHO there are options missing. For example we use formatting like this in Switzerland:

    1'234.2

What I like about this way to represent number is, that the meaning is always clear. You often see numbers like 3.421 or 4,821 and have to guess the meaning of the "," or "." – this never happens when using " ' " to group the digits as this was never uses for decimals.

What do you think, are there more formats missing?

**Comment by spacegaier** [comment]
Inside the HA frontend you should not have to guess in regards to dot vs comma, since you can define which format should be used throughout.

This of course does not invalidate your proposal to add additional formats as well.

**Comment by michaelherger** [comment]
There's hope: https://github.com/home-assistant/frontend/pull/27268 should fix this for the display. Unfortunately I haven't found whether the input can be customised at all. It seems to always use comma, even for the US where they use 1,234,567.89?... I'll keep digging.
